.github/dependabot.yml

```diff
@@ -28,6 +28,13 @@ updates:
       time: "02:00"
     open-pull-requests-limit: 10
 
+  - package-ecosystem: "cargo"
+    directory: "/cmd/oci-catalog"
+    schedule:
+      interval: "daily"
+      time: "02:30"
+    open-pull-requests-limit: 10
```

Comment:
Thanks. We've not yet released a version with oci-catalog (nor are we using it yet in the codebase), but no reason not to start getting alerts, for sure.